components/engine_rocks/src/misc.rs

```diff
@@ -321,6 +321,17 @@ impl MiscExt for RocksEngine {
         ))
     }
 
+    fn get_range_entries_and_properties(
```

Comment:
This looks like a strict superset of `get_range_entries_and_versions`. What about just replacing `get_range_entries_and_versions` with `get_range_entries_and_properties`?